dired-async.el

```diff
@@ -250,6 +255,18 @@ Same as `abort-if-file-too-large' but without user-error."
     (files--ask-user-about-large-file
      size op-type filename nil)))
 
+(defun dired-async--process-filter (proc string)
+  "`PROC' Child Emacs process filter.
+For now, it simply tries to recognise password/passphrase prompts."
+  (when (string-match dired-async-password-regexp string)
+    (process-send-string proc (concat (read-passwd string) "\n")))
+
+  ;; We must copy the string to the Child Emacs buffer anyway
```

Comment:
Why do we have to copy STRING to end of buffer.

Reply:
According to [the documentation on filter functions](https://www.gnu.org/software/emacs/manual/html_node/elisp/Filter-Functions.html), the default filter simply inserts the output of the child process into the process buffer (which is why we can see all the output of the child emacs in the buffer `*emacs*`). I think it is a good idea to imitate this behaviour after having treated the output how we wanted to.

On top of that, when we don't do it, the process buffer is not correctly killed at the end of the child process, we have the error "end of file during parsing" and the modeline still shows : [0 async job(s) running].